howso/utilities/feature_attributes/base.py

```diff
@@ -1601,13 +1634,17 @@ def _get_num_features(self) -> int:
         """Get the number of features/columns in the data."""
 
     @abstractmethod
-    def _get_num_cases(self) -> int:
-        """Get the number of cases/rows in the data."""
+    def _get_num_cases(self, feature_name: str) -> int:
```

Comment:
It looks like the relational implementation of this also needs to be updated now, since the addition of this parameter in the base class.